pygyw/layout/drawings.py

```diff
@@ -199,20 +266,18 @@ def to_commands(self) -> "list[commands.BTCommand]":
 
         ctrl_data += bytes(short_color, 'utf-8')
 
-        operations.extend([
+        return [
             # Text data
             commands.BTCommand(
                 commands.GYWCharacteristics.DISPLAY_DATA,
-                bytes(self.text, 'utf-8'),
+                bytes(line, 'utf-8'),
             ),
             # Control
             commands.BTCommand(
                 commands.GYWCharacteristics.DISPLAY_COMMAND,
                 ctrl_data,
             ),
-        ])
-
-        return operations
+        ]
```

Comment:
Can you put back `operations.extend`, or at least use operations. I know it is not really useful for now, but it feels more correct and allow us to perform some operations in the future, like ensuring the backlight is enabled, ...

Reply:
I already call super().to_commands() at line 210. Do we really need to do this for each line of text?

Reply:
Ah okey, no that's fine. My mistake